src/node/board.js

```diff
@@ -1,28 +1,61 @@
-const BoardConstruct = require('./boardConstructor');
-const BoardPrinter = require('./boardPrinter.js');
-const Point = require('./point.js');
+const constants = require('./constants.js');
+const Tile = require('./tile.js');
 
-function Board(size) {
+function Board() {
   const board = {
-    size,
-    matrix: BoardConstruct(size),
-    placePiece(point, symbol) {
-      board.matrix[point.y][point.x].setSymbol(symbol);
+    matrix: BoardConstruct(),
+    placeSymbol: (point, symbol) => {
+      board.matrix[point.x][point.y].setSymbol(symbol);
+      board.matrix[point.x][point.y].setOccupiedToTrue();
     },
-    print: () => {
+    tileIsOccupied: point => {
+      return board.matrix[point.x][point.y].isOccupied;
+    },
+    getSymbol: point => {
+      return board.matrix[point.x][point.y].symbol;
+    },
+    print() {
       BoardPrinter(board);
     },
   };
 
   return board;
 }
 
-//TEST
-const board = Board(3);
-let a = Point(1, 2);
-board.placePiece(a, 'X');
-a = Point(0, 0);
-board.placePiece(a, 'O');
-board.print();
+function BoardConstruct() {
+  const matrixOfTiles = [];
+
+  for (let i = 0; i < constants.boardSize; i += 1) {
+    const rowOfTiles = [];
+    for (let j = 0; j < constants.boardSize; j += 1) {
+      rowOfTiles.push(Tile());
+    }
+    matrixOfTiles.push(rowOfTiles);
+  }
+  return matrixOfTiles;
+}
+
+function BoardPrinter(board) {
```

Comment:
unless this `BoardPrinter` returns another function called `.print` we usually name our function by what it does:
``` js
function printBoard() {
  console.log();
}
...
printBoard();
```

when you name it `BoardPrinter`, it really seems that you created something capable of printing board